debian/upstart

```diff
@@ -0,0 +1,33 @@
+description "drserv is an authenticated HTTP API for publishing .deb package to a repo"
+
+start on runlevel [2345]
+stop on runlevel [!2345]
+
+# never stop trying to respawn
+respawn
+respawn limit unlimited
+
+# Drop privileges
+setuid drserv
+setgid nogroup
+
+env DEFAULTFILE=/etc/default/drserv
+
+pre-start script
+  [ -f $DEFAULTFILE ] && . $DEFAULTFILE
+
+  # don't start unless explicitly enabled in config
+  if [ -z "$ENABLED" ] ; then
```

Comment:
The current default of $ENABLED is `yes`, from there it is easily assumed that you can have it not enabled with `no`, testing for -z doesn't align with that assumption.